reference/tiflash/tune-performance.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Tune TiFlash Performance
+summary: Learn how to tune the performance of TiFlash.
+category: reference
+---
+
+# Tune TiFlash Performance
+
+This document introduces how to tune the performance of TiFlash, including planning machine resources, tuning TiDB parameters, and setting the size of TiKV Regions.
+
+## Plan resources
+
+If you want to save machine resources and have no requirement on isolation, you can use the method that combines the deployment of both TiKV and TiFlash. It is recommended that you save enough resources for TiKV and TiFlash respectively, and do not share disks.
+
+## Tune TiDB parameters
+
+1. For the TiDB node dedicated to OLAP/TiFlash, it is recommended that you increase the value of the [`tidb_distsql_scan_concurrency`](/reference/configuration/tidb-server/tidb-specific-variables.md#tidb_distsql_scan_concurrency) configuration item for this node to `80`:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    set @@tidb_distsql_scan_concurrency = 80;
+    ```
+
+2. Enable the optimization for TiDB Operator such as the aggregate pushdown of `JOIN` or `UNION`:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    set @@tidb_opt_agg_push_down = 1;
+    ```
+
+## Configure the size of TiKV Regions
```

Comment:
This section can be removed for now. I will pr for CN version as well. We now have an optimization for this case and region size will not be a problem in near future.

Reply:
See https://github.com/pingcap/docs-cn/pull/2686. @TomShawn

Reply:
Removed.